src/test/scala/zio/kafka/AdminSpec.scala

```diff
@@ -211,6 +215,65 @@ object AdminSpec extends DefaultRunnableSpec {
             assert(recordsAfterAltering.get(2))(isNone)
         }
       },
+      testM("create, produce, delete records from a topic with a single partition") {
```

Comment:
Could we rely on listOffsets to check the deletion instead of letting a consumer timing out?

Reply:
That's what I first tried to do, but it seems - if I correctly understood - offsets does not change

Reply:
I get it, I'll try

Reply:
@trobert  Tell me if it's better